src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import { defaultCityContentState } from '../../app/StateType'
+import switchLanguage from './switchLanguage'
+import pushCategory from './pushCategory'
+import type { CityContentStateType } from '../../app/StateType'
+import pushEvent from './pushEvent'
+import type { CityContentActionType } from '../../app/StoreActionType'
+
+export default (
+  state: CityContentStateType = defaultCityContentState, action: CityContentActionType
+): CityContentStateType => {
+  switch (action.type) {
+    case 'PUSH_CATEGORY':
+      return pushCategory(state, action)
+    case 'PUSH_EVENT':
+      return pushEvent(state, action)
+    case 'SWITCH_CITY_CONTENT_LANGUAGE': // todo:  Not working
```

Comment:
Wouldn't `SWITCH_CONTENT_LANGUAGE` be enough? Maybe `switchLanguage` should also be renamed to `switch(City)ContentLanguage`

Reply:
Will do this in NATIVE-140